docker/README.md

```diff
@@ -3,46 +3,86 @@
 
 You can build a docker image based on this Dockerfile. This image will contain only an Orion Context Broker instance, exposing port `1026`. This requires that you have [docker](https://docs.docker.com/installation/) installed on your machine.
 
-If you just want to have an Orion Context Broker running as fast as possible jump to section [The Fastest Way](#the_fastest_way).
+If you just want to have an Orion Context Broker running as fast as possible jump to section *The Fastest Way*.
```

Comment:
running as fast as possible might be confusing with regard to performance. What about 

If you just want to run Orion Context Broker right now jump to ...